packages/extension/app/v2/background.js

```diff
@@ -287,8 +287,26 @@ const automation = {
   resetBrowserTabsForNextTest (fn) {
     return Promise.try(() => {
       return browser.windows.getCurrent({ populate: true })
-    }).then((windowInfo) => {
-      return browser.tabs.remove(windowInfo.tabs.map((tab) => tab.id))
+    }).then(async (windowInfo) => {
+      let newTabId = null
+
+      try {
+        // credit to https://stackoverflow.com/questions/7000190/detect-all-firefox-versions-in-js
+        // eslint-disable-next-line no-undef
```

Comment:
I'd recommend putting `/* global window */` at the top instead of using `no-undef` here.

Reply:
addressed in [6f315a2](https://github.com/cypress-io/cypress/pull/29179/commits/6f315a2ee25b70f6f04a4c3da20fd3eac70d6437)